### PR TITLE
Release: make Homebrew release optional.

### PR DESCRIPTION
**Pull request by jelmersnoeck**
By default, we'll release to Homebrew, but if we specify it in the flag,
we will not.

### CODE REVIEW

File: cmd/release.go [hunk 1]
```diff
@@ -20,6 +20,12 @@ var release = cli.Command{
 	Usage:     "Create and publish a new release from the given tag",
 	ArgsUsage: "<tag>",
 	Action:    releaseCmd,
+	Flags: []cli.Flag{
+		cli.BoolTFlag{
+			Name:  "homebrew",
+			Usage: "Create a new Homebrew version",
+		},
```
Comment (jbowes) [inline cmd/release.go]: should the flag be --no-homebrew instead, to match your description?

Comment (jelmersnoeck) [reply cmd/release.go]: This is a True Flag, so it's true by default (if not specified). To not create a new homebrew release, you'd have to do `--homebrew=false`.

Comment (jelmersnoeck) [reply cmd/release.go]: Or did you mean the PR description?

Comment (jbowes) [reply cmd/release.go]: Nope i meant the flag, because I did not read it closely enough to see the `T` :)

**Pull request merged**